index.md

```diff
@@ -1,26 +1,22 @@
 ---
 layout: default
-title: Hank Quinlan, Horrible Cop
+title: UWBCP
 ---
 
 ## What is competitive programming?
 
 Competitive programming is a mind sport usually held over the Internet or a local network, involving participants trying to write computer programs capable of solving a set of logical or mathematical problems. Judging is based mostly upon number of problems solved and time spent for writing successful solutions, but may also include other factors (quality of output produced, execution time, program size, etc.)
 
-Competitive programming is recognized and supported by several multinational software and Internet companies, such as Google and Facebook. There are several organizations who host programming competitions on a regular basis. 
+Competitive programming is recognized and supported by several multinational software and Internet companies, such as Google and Facebook. There are several organizations who host programming competitions on a regular basis.
 
-## Who are we?
+## Upcoming Events
 
-We are a bunch of ACM officers (and volunteers) interested in spreading the love of algorithmic solving to other students in UW Bothell.
-
-## Events Held This Quarter
-
-You can find list of events held this quarter [here](insert link here). The document would be updated accordingly each quarter with new events. Stay tuned to this page!
+You can find list of events held this quarter [<span style = "color:blue">here</span>](/events.md). The document would be updated accordingly each quarter with new events. Stay tuned to this page!
 
 <ul>
   {% for post in site.posts %}
     <li>
-      <a href="/competitive-programming/{{ post.url }}">{{ post.title }}</a>
+      <a href="/mduw.github.io/{{post.url}}" style="color: blue">{{post.title}}</a>
```

Comment:
@mduw  we'll need to remove this absolute path to your accnt' for now ill take the pr.